app/controllers/users/activate_controller.rb

```diff
@@ -30,6 +30,20 @@ def create
     end
   end
 
+  def confirm_email
+    user = User.find_by(confirmation_token: params[:token])
+    if user && user.email_verified_at
+      flash[:notice] = "Votre email est déjà vérifié"
+    elsif user && user.confirmation_sent_at > 2.days.ago
```

Comment:
```suggestion
    elsif user && 2.days.ago < user.confirmation_sent_at
```

C'est con mais je trouve plus lisible